sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -143,6 +150,157 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override fun showAnnotations(
```

Comment:
We need this method. `cameraForAnnotations` will provide `cameraOptions` which users can use with their custom animations. `showAnnotations` is more like a convenience method which does animations itself using viewport (for now).

> Isn't it possible to just
> 
> map.flyTo(viewAnnotationsManager.cameraForAnnotations)

That's correct. It's much simpler and easy 😃. but I felt viewport animations are much better than simple `flyTo`. 
we can discuss if we really need to use viewport here or simple `flyTo` works. ?

Reply:
What confuses me is that we are kind of mixing the functions of different plugins, allowing ViewAnnotationManager not only to control view annotations but also the Camera.
At the same time if they work differently there indeed might be a reason to use Viewport and not direct camera animations. Thinking about the way to better expose the apis here...

Reply:
Maybe leave method for calculation of camera parameters in ViewAnnotationPlugin, and then add an extension method for animating the viewport to the ViewportPlugin? Or to map CameraOptions to OverviewViewportStateOptions?

Reply:
Yeah i guess safe option would be to use `flyTo`, since it could be possible that `viewport` might not be included in classpath (as its a different plugin) as mentioned [here](https://github.com/mapbox/mapbox-maps-android/pull/1753#discussion_r1000749223), we could although use conditional check for that.

> What confuses me is that we are kind of mixing the functions of different plugins, allowing ViewAnnotationManager not only to control view annotations but also the Camera.

even if we use `map.flyTo` we will be controling camera in viewAnnotationManager, isn't it? I feel using `viewport.transitionTo` is same as `map.flyTo`. True that viewport is a separate plugin and it might not be available in classpath.